src/app/core/services/common/integrations.service.ts

```
@@ -1,20 +1,66 @@
+    [InAppIntegration.TRAVELPERK]: '/integrations/travelperk/',
+    [InAppIntegration.INTACCT]: '/integrations/intacct',
+    [InAppIntegration.QBO]: '/integrations/qbo',
+    [InAppIntegration.SAGE300]: '/integrations/sage300',
+    [InAppIntegration.BUSINESS_CENTRAL]: '/integrations/business_central',
+    [InAppIntegration.NETSUITE]: '/integrations/netsuite',
+    [InAppIntegration.XERO]: '/integrations/xero',
+    [InAppIntegration.QBD_DIRECT]: '/integrations/qbd_direct'
+  };
+
+  getIntegrationKey(tpaName: string): IntegrationAppKey | null {
+    if (this.tpaNameToIntegrationKeyMap.hasOwnProperty(tpaName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)